### ISSUE TITLE
Return user to same Facility Locator search results state following refresh

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-cms#12867]
## Issue Description
Following the return of pagination to Facility Locator search results, we need to update the expected behavior following a page refresh. 

## Tasks
 Update behavior when page is refreshed

## Acceptance Criteria
- [ ]  When the Facility Locator page is refreshed, users should be returned to the same search results state

**Comment by laflannery** [comment]
This is actually the current behavior of search across VA.gov. I'm closing this for now and this can be updated in a wholesale approach to how search/refresh works across the site
